src/openssl.c

```diff
@@ -11990,6 +12037,8 @@ static const auxL_Reg cipher_methods[] = {
 	{ "decrypt", &cipher_decrypt },
 	{ "update",  &cipher_update },
 	{ "final",   &cipher_final },
+	{ "getTag", &cipher_get_tag },
+	{ "setTag", &cipher_set_tag },
```

Comment:
These need whitespace for alignment

Reply:
Fixed in https://github.com/wahern/luaossl/pull/201/commits/0dad2b32e96c51d0b132c1d9f5950a3be0014236